packages/components/atoms/f-form-field/stories/FormField.stories.js

```diff
@@ -81,10 +80,8 @@ export const FormFieldComponent = () => ({
             :rows="7"
             :maxlength="200"
             :labelDetails="labelDetails"
-            :assistiveText="assistiveText"
-            :has-input-description="hasInputDescription">
-                Here is a bit more text to show
-        </form-field>`
+            :labelDescription="labelDescription"
+            :assistiveText="assistiveText"/>`
 });
 
-FormFieldComponent.storyName = 'f-form-field';
+FormFieldComponent.storyName = 'Form Field';
```

Comment:
May be better as a separate PR (or raise an engineering excellence ticket for someone to pick up separately), but I think we should likely match the stories to the Figma naming and how they are split out.

So this would then be called `Text Input – Default` and then for the Affixed and Icon types, it would be `Text Input – Affixed` and `Text Input – icon`.

We should also create separate story files for dropdown, textarea and radio/checkbox components I think, as it's not obvious that those components are a part of the Default Form Field component otherwise just from looking at the nav structure – I think if I was new to the repo, it would be easy to presume that those didn't exist.

Probably enough there though to be a separate ticket in EE though!

```suggestion
FormFieldComponent.storyName = 'Default Form Field';
```

Reply:
I'll create a ticket for it